### PR TITLE
Pages with broken anchors, links etc

### PR DESCRIPTION
**Pull request by hamishwillee**

### CODE REVIEW

File: en/config/autotune.md [hunk 1]
```diff
@@ -98,11 +98,11 @@ Additional notes:
 #### The drone oscillates when performing the testing maneuvers prior to the auto-tuning
 
 * slow oscillations (1 oscillation per second or slower): this often occurs on large platforms and means that the attitude loop is too fast compared to the rate loop.
-   - **Multicopter:** decrease [MC_ROLL_P](../advanced_config/parameter_reference.md#MC_ROLL_P) and [MC_PITCH_P](../advanced_config/parameter_reference.md#MC_PITCH_P) by steps of 1.0
-   - **Fixed-wing:** increase [FW_R_TC](../advanced_config/parameter_reference.md#FW_R_TC), [FW_P_TC](../advanced_config/parameter_reference.md#FW_P_TC), [FW_Y_TC](../advanced_config/parameter_reference.md#FW_Y_TC) by steps of 0.1
+   - **Multicopter:** decrease [MC_ROLL_P](../advanced_config/parameter_reference.md#MC_ROLL_P) and [MC_PITCH_P](../advanced_config/parameter_reference.md#MC_PITCH_P) by steps of 1.0.
+   - **Fixed-wing:** increase [FW_R_TC](../advanced_config/parameter_reference.md#FW_R_TC), [FW_P_TC](../advanced_config/parameter_reference.md#FW_P_TC), [FW_Y_TC](../advanced_config/parameter_reference.md#FW_Y_TC) by steps of 0.1.
```
Comment (hamishwillee) [inline en/config/autotune.md:102]: 'FW_Y_TC', 'FW_RR_R',  'FW_RR_Y' parameters referenced in autotune do not appear to exist. @MaEtUgR @bresch can either of you suggest the required change? (and PR it).

File: en/config_fw/trimming_guide_fixedwing.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ For this purpose, a bilinear curve (see figure below) function of airspeed and a
 <!-- The drawing is on draw.io: https://drive.google.com/file/d/15AbscUF1kRdWMh8ONcCRu6QBwGbqVGfl/view?usp=sharing
 Request access from dev team. -->
 
-A perfectly symmetrical airframe would only require pitch trim increments, but since a real airframe is never perfectly symmetrical, roll and yaw trims increments are also sometimes required. 
+A perfectly symmetrical airframe would only require pitch trim increments, but since a real airframe is never perfectly symmetrical, roll and yaw trims increments are also sometimes required.
 
 :::note
 If a scale factor different from 1.0 is used for the flaps (parameter [FW_FLAPS_SCL](../advanced_config/parameter_reference.md#FW_FLAPS_SCL)), the trim increment added by the `FW_DTRIM_R/P_FLPS`is also scaled by the same factor.
```
Comment (hamishwillee) [inline en/config_fw/trimming_guide_fixedwing.md:55]: @sfuhrer FW_FLAPS_SCL param seems to have gone. There now appears to be FW_FLAPS_LND_SCL  and FW_FLAPS_TO_SCL . What needs to happen here?

PS Any chance you have a timeline on updating for the NAV_GPSF_ params that were removed?

**Comment by hamishwillee** [comment]
Merging fixes. There are open questions for dev team in this.

File: en/config_fw/trimming_guide_fixedwing.md (hunk 2 above)
Comment (sfuhrer) [reply en/config_fw/trimming_guide_fixedwing.md:55]: >@sfuhrer FW_FLAPS_SCL param seems to have gone. There now appears to be FW_FLAPS_LND_SCL and FW_FLAPS_TO_SCL . What needs to happen here?

I think it's fine to just remove the note, as it no longer applies with FW_FLAP_SCALE gone.

Comment (sfuhrer) [reply en/config_fw/trimming_guide_fixedwing.md:55]: btw found a tipo: https://github.com/PX4/PX4-user_guide/commit/62d185424410591697a30655a774bb7ba6e04c1b

Comment (sfuhrer) [reply en/config_fw/trimming_guide_fixedwing.md:55]: >PS Any chance you have a timeline on updating for the NAV_GPSF_ params that were removed?

Thanks for the nudge. I found out that it is currently partly broken on main, and didn't find alignment on how to fix it. https://github.com/PX4/PX4-Autopilot/pull/19942

Anyway, here the proposal for the updated docs (that's how it should be and hopefully is soon again). 
https://github.com/PX4/PX4-user_guide/pull/2006

Comment (hamishwillee) [reply en/config_fw/trimming_guide_fixedwing.md:55]: Thanks! Fixed by https://github.com/PX4/PX4-user_guide/commit/aa5163a16884b6b1f54c4ae5f2235468de880f10

**Pull request merged**